src/structures/GuildMember.js

```diff
@@ -121,8 +121,7 @@ class GuildMember extends Base {
    * @readonly
    */
   get lastMessage() {
-    const channel = this.guild.channels.cache.get(this.lastMessageChannelID);
-    return (channel && channel.messages.cache.get(this.lastMessageID)) || null;
+    return this.guild.channels.resolve(this.lastMessageChannelID)?.messages.resolve(this.lastMessageID) ?? null;
```

Comment:
That's redundant:
```suggestion
    return this.guild.channels.resolve(this.lastMessageChannelID)?.messages.resolve(this.lastMessageID);
```

Reply:
Am I wrong or with the optional chaining operator it's possible that this returns undefined?

Reply:
No, you are actually correct. 😅